msal/application.py

```diff
@@ -68,6 +69,16 @@ def extract_certs(public_cert_content):
 
 class ClientApplication(object):
 
+    ACQUIRE_TOKEN_SILENT_ID = "84"
+    ACQUIRE_TOKEN_BY_USERNAME_PASSWORD_ID = "301"
+    ACQUIRE_TOKEN_ON_BEHALF_OF_ID = "523"
+    INITIATE_DEVICE_FLOW = "621"
+    ACQUIRE_TOKEN_BY_DEVICE_FLOW_ID = "622"
+    ACQUIRE_TOKEN_FOR_CLIENT_ID = "730"
+    ACQUIRE_TOKEN_BY_AUTHORIZATION_CODE_ID = "832"
+    GET_ACCOUNTS_ID = "902"
+    REMOVE_ACCOUNT_ID = "903"
```

Comment:
get_accounts and remove_account id not being used at the moment, but will be when we include a solution for last request header

Reply:
Link to API ID doc: https://microsoft.sharepoint.com/:w:/t/aad/devex/EZGF46opilBKrgC59mUFttIBebPBY6MMGLO1eGz0uXQh6w?e=TSq76f

Note that these values are not in there yet. The way that this has worked in the past for .NET and Java is that we have picked values that fit within the ranges specified which have not been taken, then we'd update the doc. 

We are working with some of the other teams to make sure that most up to date values are in there. For that, we have this spreadsheet: https://microsoft-my.sharepoint-df.com/:x:/p/sagonzal/EXSrr4vM1utAqQQfD6bMln4BYKwjrqh3cagiNJWPVNjLzw?e=G5FybL